docs/releases/1.8.3.txt

```diff
@@ -74,3 +74,6 @@ Bugfixes
 
 * Fixed crash when uploading images with MIME types that Pillow doesn't detect,
   such as bitmap, in ``forms.ImageField`` (:ticket:`24948`).
+
+* Fixed a regression when deleting a model that has a ``GenericRelation`` and a
```

Comment:
"deleting in a model through the admin" doesn't make sense in my ears :stuck_out_tongue_winking_eye: